osu.Game/Screens/Edit/Compose/Components/Timeline/TimelineHitObjectBlueprint.cs

```diff
@@ -34,11 +39,21 @@ public class TimelineHitObjectBlueprint : SelectionBlueprint
 
         private readonly List<Container> shadowComponents = new List<Container>();
 
+        private DrawableHitObject drawableHitObject;
+
+        private Bindable<Color4> comboColour;
+
+        private readonly Container mainComponents;
+
+        private readonly OsuSpriteText comboIndexText;
+
+        private Bindable<int> comboIndex;
+
         private const float thickness = 5;
 
         private const float shadow_radius = 5;
 
-        private const float circle_size = 16;
+        private const float circle_size = 24;
```

Comment:
Not sure if this was a conscious design choice but I feel at this size the numbers are pretty hard to see properly. How about something like 32? Although I do admit that size looks pretty silly on combo-index-less rulesets...

Reply:
May need to change the font, not sure. Don't want to go bigger in size though, looks wrong even on osu! ruleset.